utils/generate-command-code.py

```diff
@@ -242,7 +242,8 @@ def write_internal_structs(self, f):
 
 def to_c_name(str):
     return str.replace(":", "").replace(".", "_").replace("$", "_")\
-        .replace("^", "_").replace("*", "_").replace("-", "_")
+        .replace("^", "_").replace("*", "_").replace("-", "_") \
+        .replace("\\d", "_").replace("+", "_").replace("\\", "_")
```

Comment:
If we replace `\` why do we need to also replace the `d`?

Reply:
yes, there is no need for d.